<?xml version="1.0" ?>

<launch>    

    <param name="gazebo"                   value="true"     type="bool"/>
    <param name="gazebo_robot_name"        value="alice_robot" />

    <param name="offset_file_path"             value="$(find alice_manager)/config/offset.yaml"/>
    <param name="robot_file_path"          value="$(find alice_manager)/config/alice.robot"/>
    <param name="init_file_path"           value="$(find alice_manager)/config/dxl_init.yaml"/>
      
    <!-- alice Manager -->
    <node name="alice_manager_node" pkg="alice_manager" type="alice_manager_node" output="screen"/>
    <!--<node name="decision_module" pkg="decision_module" type="decision_module" output="screen"/>-->
    
</launch>

